src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -12,11 +12,25 @@ public Elevator(ElevatorIO io) {
     this.io = io;
   }
 
+  /** 
+   * sets the position of the elevator
+   * 
+   * @param position position of elevator, from -1.0 to 1.0
```

Comment:
this is incorrect, the position is in rotations, bounded by the elevators max and min height

Reply:
mb

Reply:
is it possible to use variables in docstrings? if so how do I do that?

Reply:
nope

Reply:
but you can link other methods or do @value to define a value

Reply:
so ill just say in meters, from 0 to X for now?

Reply:
id just say from minimum elevator height to max height